adr/001-hosting.md

```diff
@@ -0,0 +1,93 @@
+# Hosting
+
+* Status: proposed
+* Deciders: [Shawn Allen](https://github.com/shawnbot), [HuiRan Shao](https://github.com/hshaosf)
+* Date: 2022-03-01
+
+## Context and Problem Statement
+
+The archive web server will primarily serve as a [URL redirect] engine for legacy sites. It will need to:
+
+1. Accept HTTP requests from multiple domains (respecting the `Host` header)
+2. Redirect known URLs to their new homes on [sf.gov][] (explicit redirects)
+3. Redirect all other document URLs (HTML, PDF, Word, Excel, etc.) to their archived snapshots
+4. Serve helpful [404] responses to all other URLs
+
+From an engineering standpoint, we also expect to:
+
+* Easily run, develop, and test the server locally
+* Run tests locally, in CI, and in review environments
+* Have alerts that notify us when problems arise
+* Have easily accessible logs to diagnose problems
+* Scale the web server to meet high traffic demands
+
+We need to choose a hosting platform that serves these needs.
+
+## Considered Options
+
+* [Azure functions](#azure-functions)
+* [Heroku](#heroku)
+* [Pantheon](#pantheon)
+
+## Decision Outcome
+
+Chosen option: **Heroku**
```

Comment:
```suggestion
Propose decision: **Heroku**
```
Sounds good. Then maybe we rename this to propose then create separate PR once the decision is made?